subcommands/targets/offline-update.go

```diff
@@ -35,15 +37,19 @@ var (
 	ouTufOnly              bool
 	ouNoApps               bool
 	ouAllowMultipleTargets bool
+	ouWave                 string
 )
 
 func init() {
 	offlineUpdateCmd := &cobra.Command{
-		Use:   "offline-update <target-name> <dst> --tag <tag> [--prod] [--expires-in-days <days>] [--tuf-only]",
+		Use:   "offline-update <target-name> <dst> --tag <tag> [--prod | --wave <wave-name>] [--expires-in-days <days>] [--tuf-only]",
 		Short: "Download Target content for an offline update",
 		Run:   doOfflineUpdate,
 		Args:  cobra.ExactArgs(2),
 		Example: `
+	# Download update content of the wave target #1451 for "intel-corei7-64" hardware type
```

Comment:
I'm not sure how to word this, but I think this command is getting complicated enough that we kind of need to briefly describe the "why" of this option and maybe the others so that people can figure out what options they need.

Reply:
Not sure I understand. I think a customer is the one who should decide on "why" - whether to compose an offline bundle for wave, or prod, or CI target, we just explain mechanics to do accomplish what they decided.

Reply:
maybe its just content that belongs in docs, but we have all this stuff floating around: tags, CI, prod, and now wave. It could be intimidating for a regular user to navigate this. i'm fine ignoring this

Reply:
I think explanation should go the user doc. Not sure I see a good way to explain all that in the command comment.